src/ng-at-internet.config.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+angular.module("ng-at-internet")
+    .config(function ($provide, atInternetProvider) {
+        $provide.decorator("atInternet", function ($delegate) {
```

Comment:
You should test decorator behavior
- No track before config is set
- Everything is tracked after config is set
- Exception raised after 30 tracks without config